### PR TITLE
Publish test results

### PR DESCRIPTION
**Pull request by kerbou**
Publish test results after running testsuite

Reference: https://app.zenhub.com/workspaces/the-outlaws-6193fe815d79fc0011e741b1/issues/gh/energinet-datahub/the-outlaws/997

Test run results for geh-core using `dotnet-solution-build-and-test`:
https://github.com/Energinet-DataHub/geh-core/actions/runs/4542914200

Test run results for PR in opengeh-edi using `dotnet-postbuild-test` workflow:
https://github.com/Energinet-DataHub/opengeh-edi/actions/runs/4553198807

Test run result when PR is merged to main and `dotnet-solution-ci` is invoked to generate CodeCov reports:
https://github.com/Energinet-DataHub/opengeh-edi/actions/runs/4554571728

### CODE REVIEW

File: .github/workflows/dotnet-postbuild-test.yml [hunk 1]
```diff
@@ -279,6 +282,16 @@ jobs:
           $testFailures = Select-String -Path '.\TestResults\logs.trx' -Pattern '\[FAIL\]'
           $testFailures
 
+      - name: Publish test report
+        if: always()
+        uses: EnricoMi/publish-unit-test-result-action/composite@v2
+        with:
+          check_name: Test results for ${{ inputs.TESTS_DLL_FILE_PATH }}
+          comment_title: Test results for ${{ inputs.TESTS_DLL_FILE_PATH }}
```
Comment (Renetnielsen) [inline .github/workflows/dotnet-postbuild-test.yml:290]: Inputs are lowercase now

**Review by Renetnielsen — CHANGES_REQUESTED** [review]
Many inputs are using UPPERCASE

File: .github/workflows/dotnet-postbuild-test.yml [hunk 2]
```diff
@@ -279,6 +282,16 @@ jobs:
           $testFailures = Select-String -Path '.\TestResults\logs.trx' -Pattern '\[FAIL\]'
           $testFailures
 
+      - name: Publish test report
+        if: always()
+        uses: EnricoMi/publish-unit-test-result-action/composite@v2
+        with:
+          check_name: Test results for ${{ inputs.TESTS_DLL_FILE_PATH }}
```
[the remark this answers is not in the document]
Comment (kerbou) [reply .github/workflows/dotnet-postbuild-test.yml:289]: Fixed

File: .github/workflows/dotnet-postbuild-test.yml (hunk 1 above)
Comment (kerbou) [reply .github/workflows/dotnet-postbuild-test.yml:290]: Fixed

**Review by Renetnielsen — APPROVED** [review]

**Pull request merged**